example/config/managedk8s.yaml

```
@@ -10,6 +10,24 @@ providers:                   # contains information about known providers
     name: DISA Kubernetes Security Technical Implementation Guide
     version: v1r11
     ruleOptions:
+    # - ruleID: "242383"
+    #   args:
+    #     acceptedResources:
+    #    - selectResource:
```

Can we remove the selectResource? It seems that acceptedResources can directly be an array that accepts the kind, apiVersion, etc
```suggestion
    #    - selectResource:
```